catalyst/utils/torch.py

```diff
@@ -1,8 +1,7 @@
-from typing import Dict, Iterable, List, Tuple, Union  # isort:skip
 import collections
-import copy
 import os
 import re
+from typing import Dict, Iterable, List, Union
```

Comment:
Could you please return # isort:skip and move from typing import ... line to the top of the file?

Reply:
done